### PR TITLE
[Proposal] Add contributions section 

### PR DESCRIPTION
**Pull request by Ilithy**
Simple proposal to add a _(small)_ section for contributions.

Do not hesitate to refuse if it does not suit. 

Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,15 @@ With AOSP / LineageOS:
...
 ## Get help
 
-Open an issue [on github](https://github.com/2bllw8/anemo/issues/)
+Open an discussion [on github](https://github.com/2bllw8/anemo/discussions/new?category=Q-A)
+
+## Contributions
+
+**Every contributions, including ideas, feature requests, bug reports and Pull Requests are welcome!**
+
+- **solve bug(s)**, add **features** or **translate** Anemo to your native language by making a **[pull request](https://help.github.com/articles/about-pull-requests/)**
```
Comment (2bllw8) [inline README.md:53]: Upper case S

File: README.md [hunk 2]
```diff
@@ -44,7 +44,15 @@ With AOSP / LineageOS:
...
+- You have an idea for improvement or a new feature but don't know how to code it, open a **[feature request](https://github.com/2bllw8/anemo/issues/new?assignees=&labels=enhancement&template=feature_request.yml&title=[Feature+request]+)**
```
Comment (2bllw8) [inline README.md:54]: Replace with
```markdown
You have an idea for improvement or a new feature? Open a...
```

Comment (2bllw8) [inline README.md:54]: Replace with
```markdown
You have an idea for improvement or a new feature? Open a...
```

Comment (Ilithy) [reply README.md]: I have made the changes, let me know if it is ok, thanks

**Review by 2bllw8 — APPROVED** [review]

**Pull request merged**